### ISSUE TITLE
fetcher sheets have an empty data on index 0

### ISSUE THREAD

**Issue by mazipan** [kawalcovid19/wargabantuwarga.com#243]
## Description

Based on #242, you will know that we have an empty data on `/data/wbw-sheets.json` produced by script on `/etc/fetchers/fetch-sheets.ts`.

Sample empty data:

```json
[
  {"id":"358239346","name":"Tentang Database","data":[]},
  {
    "id": "0",
    "name": "DKI Jakarta",
    "data": []
  },
]
```

## Expected Behaviour

Remove "Tentang Database" from the result.

## Additional Information

Need to check, is our App already expect this condition, if yes, you need to change it also

**Comment by ekamuktia** [comment]
one of the part that will be affected by the change
https://github.com/kawalcovid19/wargabantuwarga.com/blob/4851da40bce0b5b28acdc77fc4e1209ffdddba00/pages/provinces/index.tsx#L63

**Comment by zainfathoni** [comment]
As far as I can remember, that was intentional, due to a bug caused by the non-existence of that first entry. Can you recall what it was @adityapurwa?
We have to make sure that the bug was taken care of before making this change.

**Comment by adityapurwa** [comment]
Hello hello, so from what I understand. Do we want to explicitly exclude the first sheet from the result? @zainfathoni, @ekamuktia mentioned that it will affect the file mentioned above.

**Comment by zainfathoni** [comment]
I was talking about #141 PR @adityapurwa.
That was the reason why we included the first sheet in the fetch results.
The risk should be manageable, isn't it?

**Comment by mazipan** [comment]
If the performance of scrapping the Google Sheet was the main issue, we don't need to change it, we just need to remove the first array after finish the scrapping process.

It's not high cost in my opinion

**Comment by adityapurwa** [comment]
I assume if we're removing the first sheet after fetching, the performance cost would not be that high.

If the others agreed to remove the first sheet, then we can do it.

🚀

**Comment by Namchee** [comment]
This issue should have been resolved already by #361. The first sheet is removed after the sheets have been fetched.
